gradle.properties

```diff
@@ -1,5 +1,5 @@
 org.gradle.caching=true
 systemProp.org.gradle.internal.publish.checksums.insecure=true
 version=3.2.7-SNAPSHOT
-discordJsonVersion=1.6.18
+discordJsonVersion=1.6.19-SNAPSHOT
```

Comment:
```suggestion
discordJsonVersion=1.6.18
```

Reply:
If i make that i broke the PR build check, when the PR in json is merged i can edit that here.